components/schema-migrator/README.md

```diff
@@ -0,0 +1,9 @@
+# Schema Migrator
+
+## Overview
+
+The Schema Migrator is responsible for database schema migrations.
+
+## Development
+
+If you want to modify database schema used by compass, add migration files (follow [this](https://github.com/golang-migrate/migrate/blob/master/MIGRATIONS.md) instructions) to `migrations` directory and bump component version value in compass chart.
```

Comment:
please write explicitly that new image of migrator will be produced that contains all migration files